wiki/osu!_wiki/en.md

```diff
@@ -0,0 +1,41 @@
+
+## Structure
+
+The gateway to the osu! wiki is the [main page](/wiki/Main_Page), which is divided into several high-level categories, each having a general description and including a list of important articles. The articles not listed on the main page, which are typically more specific, can be reached either from other pages using inline links, or through the search feature of the website.
+
+An individual article describes a single term or concept and gives a general overview on adjacent topics. To better illustrate the subject, it is divided into subsections covering different parts of the term. It may also include relevant screenshots or other visual explanations, or link to other articles on topic<!-- TODO: via the use of infoboxes. uncomment this when they are ready (https://github.com/ppy/osu-wiki/issues/5440) -->.
+
+For most articles, translations are available. They are done by the community members themselves, and the choice of language is only limited by a [list of languages](/wiki/Article_styling_criteria/Formatting#locales) supported by the website itself.
+
+## Model
+
+The osu! wiki is built around the concept of accessibility: edits and translations are encouraged and may be done by anyone capable and knowledgeable. The data for articles is stored in a [GitHub repository](https://github.com/ppy/osu-wiki) and updated by contributors through [use of pull requests](/wiki/osu!_wiki_Contribution_Guide), which then are reviewed and merged by the [wiki maintainers](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
+
+The articles are formatted using [GitHub-flavoured Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Additionally, the website engine implements custom features, which are crucial for the wiki as a whole, such as redirects, front matter, or formatting improvements<!-- TODO: article on custom features of the wiki? it could help us ease the ASC a bit as well -->.
+
+Articles are composed according to the [styling criteria](/wiki/Article_styling_criteria) of the osu! wiki, which means they are written in a neutral language and follow common formatting and design rules. Some informational articles serve as a list of commonly asked questions (FAQ) on the topic and use informal language. Translations follow the content parity principle and comply with the same formatting and writing rules.
```

Comment:
perhaps it's best to use the actual term when also including its acronym

also note that other languages can have different grammar and punctuation rules. some formatting things are global (like markdown), but i'm pretty sure the writing part is locale-specific

```suggestion
Articles are composed according to the [styling criteria](/wiki/Article_styling_criteria) of the osu! wiki, which means they are written in a neutral language and follow common formatting and design rules. Some informational articles serve as a list of frequently asked questions (FAQ) on the topic and use informal language. Translations follow the content parity principle and comply with the same formatting rules.
```

Reply:
absolutely. what I was referring to is *some* parts of `Writing`, like the tone, but that probably would be and odd thing to cover here